components/Sidebar/Sidebar.tsx

```diff
@@ -16,16 +16,21 @@ interface SidebarProps {
     main: React.ReactNode;
     sheet?: React.ReactNode & { props: SidebarSheetProps };
     footer?: React.ReactNode;
+    collapse?: boolean;
 }
 
 export class Sidebar extends React.Component<SidebarProps> {
+    public static defaultProps: Partial<SidebarProps> = {
+        collapse: false,
+    };
+
     static AppLogo = AppLogo;
     static Item = SidebarItem;
 
     nav: SidebarBehavior;
 
     componentDidMount() {
-        this.nav = new SidebarBehavior();
+        this.nav = new SidebarBehavior(this.props.collapse || false);
```

Comment:
Just a suggestion, in the constructor we could do:

```
constructor(props: SidebarProps) {
    super(Object.assign({}, Sidebar.defaultsProps, props);
}
```

So that we rely on `defaultProps` definitions.
We could use that as a commn pattern.